src/Humanizer.Tests.Shared/CollectionHumanizeTests.cs

```diff
@@ -87,5 +88,58 @@ public void HumanizeUsesObjectFormatterWhenSeparatorIsProvided()
             var humanized = _testCollection.Humanize(sc => string.Format("SomeObject #{0} - {1}", sc.SomeInt, sc.SomeString), "or");
             Assert.Equal("SomeObject #1 - One, SomeObject #2 - Two, or SomeObject #3 - Three", humanized);
         }
+
+        [Fact]
+        public void HumanizeHandlesNullItemsWithoutAnException()
+        {
+            new object[] { null, null }.Humanize();
```

Comment:
Okay. How do I add a check that an exception is not thrown, other than exactly what I have there?